File: app_pojav_zh/src/main/java/com/movtery/pojavzh/ui/subassembly/customcontrols/ControlItemBean.java
package com.movtery.pojavzh.ui.subassembly.customcontrols;

public class ControlItemBean {
    private ControlInfoData controlInfoData;
    private boolean isHighlighted = false;

    public ControlItemBean(ControlInfoData controlInfoData) {
        this.controlInfoData = controlInfoData;
    }

    public ControlInfoData getControlInfoData() {
        return controlInfoData;
    }

    public void setControlInfoData(ControlInfoData controlInfoData) {
        this.controlInfoData = controlInfoData;
    }

    public boolean isHighlighted() {
        return isHighlighted;
    }

    public void setHighlighted(boolean highlighted) {
        isHighlighted = highlighted;
    }
}

File: app_pojav_zh/src/main/java/com/movtery/pojavzh/ui/subassembly/customcontrols/ControlItemBean.kt
package com.movtery.pojavzh.ui.subassembly.customcontrols;

public class ControlItemBean {
    private final ControlInfoData controlInfoData;
    private boolean isHighlighted = false;
    private boolean isInvalid = false;

    public ControlItemBean(ControlInfoData controlInfoData) {
        this.controlInfoData = controlInfoData;
    }

    public ControlInfoData getControlInfoData() {
        return controlInfoData;
    }

    public boolean isHighlighted() {
        return isHighlighted;
    }

    public void setHighlighted(boolean highlighted) {
        isHighlighted = highlighted;
    }

    public boolean isInvalid() {
        return isInvalid;
    }

    public void setInvalid(boolean invalid) {
        isInvalid = invalid;
    }
}
